locales/pdk.pot

```diff
@@ -6,11 +6,11 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: puppet development kit v1.3.1-7-g6c96455\n"
+"Project-Id-Version: puppet development kit v1.3.2-69-g30c8e60\n"
```

Comment:
Seems like this should be 1.4.0?

Reply:
Yeah, this is weird...

Reply:
this is likely based off `git describe`, not lib/pdk/version.rb, so \*shrug*

Reply:
Yeah, I don't think it actually matters